dotcom-rendering/src/web/components/SecureSignupIframe.importable.tsx

```diff
@@ -283,6 +283,58 @@ export const SecureSignupIframe = ({
 		resetIframeHeight();
 	};
 
+	const addFontToIframe = (requiredFontNames: string[]) => {
+		const { current: iframe } = iframeRef;
+
+		// fallback: clone the whole fonts node into the iframe.
+		// used if the browser does not support FontFaceSet.add
+		const fallbackMethod = () => {
+			const fontNode = document
+				.querySelector('style.webfont')
+				?.cloneNode(true);
+
+			if (fontNode) {
+				iframe?.contentDocument?.head.appendChild(fontNode);
+			}
+		};
+
+		const iframeFontFaceSet = iframe?.contentDocument?.fonts;
+		if (!iframeFontFaceSet) {
+			return fallbackMethod();
+		}
+
+		// The ts.dom interface for FontFaceSet does not contain the .add method,
+		// browser support is not universal.
```

Comment:
i think this is only true for IE, which we do not need to support anymore. maybe an alternative approach is to use `.add` where supported, and fallback to system fonts where not?

Reply:
sounds reasonable - the function can just exit if the method isn't supported